lighty-modules/lighty-gnmi/lighty-gnmi-sb/README.md

```diff
@@ -55,3 +63,102 @@ This encoding enforcement does not apply to SetRequest/GetResponse targeting sca
 As stated above, only JSON_IETF is supported for encoding structured data types. This means that device MUST declare it's
  support of JSON_IETF encoding in [CapabilityResponse](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#322-the-capabilityresponse-message)
   supported_encodings field. If JSON_IETF is not present in supported_encoding field, connection of gNMI device is closed.
+
+##lighty.io gNMI south-bound with lighty.io Controller example
+This example will show how to programmatically connect lighty gNMI-sb with gNMI device. First we start lighty gGNMI,
+lighty controller and gNMI device. For device in this example will be used gNMI device simulator. Creating device 
+connection and All CRUD operation will be performed with writing data directly to MD-SAL data store
+without using RESTCONF. Full example can be found inside test [GnmiWithoutRestconfTest](../lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/GnmiWithoutRestconfTest.java).
+
+1. Initialize lighty Controller lighty gNMI.
+```java
+        lightyController = new LightyControllerBuilder()
+                .from(ControllerConfigUtils.getConfiguration(Files.newInputStream(CONFIGURATION_PATH)))
+                .build();
+        lightyController.start().get();
+
+        gnmiSouthboundModule = new GnmiSouthboundModuleBuilder()
+                .withConfig(GnmiConfigUtils.getGnmiConfiguration(Files.newInputStream(CONFIGURATION_PATH)))
+                .withLightyServices(lightyController.getServices())
+                .withExecutorService(Executors.newCachedThreadPool())
+                .withEncryptionService(createEncryptionService())
+                .build();
+        gnmiSouthboundModule.start().get();
+```
+
+2. Connect gNMI device
```

Comment:
Reference our gNMI simulator.  e.g ..(For testing purposes, one can use ours gNMI simulated device [LINK] ).

Reply:
done, here:
https://github.com/PANTHEONtech/lighty/pull/737/files#diff-a2d889e78f690ac2a98fb529411b2cf817f0a4dbfbe4d66d7da066fc21fae0b3R70